packages/opentelemetry-exporter-collector/src/transformMetrics.ts

```diff
@@ -172,6 +170,31 @@ export function toHistogramPoint(
   };
 }
 
+/**
+ * Returns a SummaryPoint to the collector
+ * @param metric
+ * @param startTime
+ */
+export function toSummaryPoint(
+  metric: MetricRecord,
+  startTime: number
+): opentelemetryProto.metrics.v1.SummaryDataPoint {
+  const distValue = metric.aggregator.toPoint().value as Distribution;
+  return {
+    labels: toCollectorLabels(metric.labels),
+    sum: distValue.sum,
+    count: distValue.count,
+    startTimeUnixNano: startTime,
+    timeUnixNano: core.hrTimeToNanoseconds(
+      metric.aggregator.toPoint().timestamp
```

Comment:
Nit: you are calling `metric.aggregator.toPoint()` twice consider caching this into variable.